data/scripts.js

```diff
@@ -564,6 +564,7 @@ exports.BattleScripts = {
 	moveHit: function (target, pokemon, move, moveData, isSecondary, isSelf) {
 		let damage;
 		move = this.getMoveCopy(move);
+		move.zBrokeProtect = false;
```

Comment:
The question is, what other move state should get reset?

Reply:
Literally all of it? We could `getMoveCopy(toId(move))`

Reply:
Most state gets set in `ModifyMove`, near the beginning of `useMove`, and it is expected to apply to all targets of spread moves. I looked for all the possible move states I could find and this was the only one that I found that was hit-specific.

Reply:
I think there were some other volatiles that got converted to move flags; those will need to be taken a look at at some later point.

Maybe move all of this into `move.hitFlags`?

Reply:
Actually I already looked at those move states too.